tests/std/tests/P2502R2_generator_promise/test.cpp

```diff
@@ -22,6 +22,16 @@
 
 using namespace std;
 
+#pragma warning(disable : 28251) // Inconsistent annotation for 'new[]': this instance has no annotations.
+
+void* operator new[](size_t) {
+    unreachable();

```

Comment:
Better `abort()` to crash on reaching rather than UB on reaching. Same below.
(Include `<cstdlib>`)